cmd/main.go

```diff
@@ -42,12 +45,14 @@ func main() {
 	accountStore := account_postgres.NewRepository(db)
 	transStore := transfers_postgres.NewRepository(db)
 	billStore := bills_postgres.NewRepository(db)
+	activitiesDb := activities_postgres.NewRepository(db)
 
 	accUsecase := account.NewAccountUsecase(accountStore)
 	transferStore := transfers.NewTransferUsecase(transStore, accUsecase)
 	blStore := bill.NewBillUseCase(billStore, accUsecase)
+	activityStore := activities.NewAccountActivityUsecase(activitiesDb)
```

Comment:
Acho q faz mais sentido chamar de usecase, não acha? Ficaria tipo a linha 50
```suggestion
	activityUsecase := activities.NewAccountActivityUsecase(activitiesDb)
```